en/components/list.md

```diff
@@ -5,71 +5,368 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
+    phone: "573-394-9254"
+}];
+```
 
-#### Methods
-- `addChild` - Add `IListChild` component to children array and to the respective specific array
-- `removeChild` - Remove `IListChild` component from children array and from the respective specific array
+Now that we have some data we want to render, let's set up some markup:
 
-#### Events
-- `onPanStateChange` - Triggered when pan gesture is executed on list item
-- `onLeftPan` - Triggered when left pan gesture is executed on list item
-- `onRightPan` - Triggered when right pan gesture is executed on list item
-<div class="divider--half"></div>
+```html
+<!--contacts.component.html-->
+
+<igx-list>
+  <igx-list-item isHeader="true">
+    Contacts
+  </igx-list-item>
+  <igx-list-item *ngFor="let contact of contacts">
+    <span class="name">{{ contact.name }}</span>
+    <span class="phone">{{ contact.phone }}</span>
+  </igx-list-item>
+</igx-list>
+```
+
+> [!NOTE]
+> The list item uses `flex` as its display value, with `flex-direction` set to `column`. Bear this in mind when building list layouts.
+
+You may have noticed that despite the fact that we used span elements to display the name and phone number for our contacts, we still see them rendered one under the other. This is due to the column nature of each list item. Now that we have that out of the way, let's add some custom styling. We added two new classes to our name and phone spans - _name_ and _phone_. Let's use those classes to style the items:
+
+```css
+/* contacts.component.css */
+
+.name {
+    font-weight: 600;
+}
+
+.phone {
+    font-size: 0.875em;
+}
+```
+
+After all that our list should now look like that:
+
+<div class="sample-container" style="height: 400px">
+<iframe src='http://{environment:host}/list-sample-3' width="100%" height="100%" seamless frameBorder="0"></inframe>
+</div>
+
+#### Adding Avatar and Icons
+
+We can use some of our other components in conjuction with the Igx List component to enrich the experience and add some functionality. We can have a nice picture avatar to the left of the name and phone values. Additionally, we can add a star icon to the right of them to allow the user to favorite a contact. To do that let's grab the [**IgxAvatar**](https://www.infragistics.com/products/ignite-ui-angular/angular/components/avatar.html) and [**IgxIcon**](https://www.infragistics.com/products/ignite-ui-angular/angular/components/icon.html) modules and import them in our **app.module.ts** file.
+
+```typescript
+// app.module.ts
+
+...
+import {
+    IgxListModule,
+    IgxAvatarModule,
+    IgxIconModule
+} from 'igniteui-js-blocks/main';
+
+@NgModule({
+    ...
+    imports: [..., IgxAvatarModule, IgxIconModule],
+})
+export class AppModule {}
+```
+
+Next, we need to add some more information to our contact object, like a `photo` source for our avatar and a `isFavorite` property to indicate the contact's favorite status.
+
+```typescript
+// contacts.comoponent.ts
+
+private contacts = [{
```

Comment:
Change it to `public`. 
Any decorated class members and data bound properties that are not `public` will currently break AOT builds